tests/libscrypt/Makefile

```diff
@@ -0,0 +1,4 @@
+.POSIX:
+
+all:
+	${CC} sample-libscrypt.c -lscrypt
```

Comment:
I forgot that I gave a sample compile command in the README.  We probably don't need this makefile, then.